### DISCUSSION TITLE
What is the recommended way to add Alpinejs components when using @fluxScripts?

### DISCUSSION THREAD

**Discussion by rikh42** [livewire/flux#672] [Feature Requests] [answered]
When using flux with @fluxStyles and @fluxScripts, is it possible to add custom Alpine.js components? 

Specifically, in my app.js, how do I access the Alpine instance created by flux, so I can do something like `Alpine.data('example', example)`?

Before flux, I was using the method described in the LiveWire docs at https://livewire.laravel.com/docs/alpine#manually-bundling-alpine-in-your-javascript-build. 

How do I manage this when using Flux and should there be something in the docs to cover this, as there is in Livewire?

**Comment by jeffchown** [comment, marked as answer]
@rikh42 I still use the method described in https://livewire.laravel.com/docs/alpine#manually-bundling-alpine-in-your-javascript-build with Flux and everything works as expected (re adding custom Alpine.js components, etc.).

**Reply by rikh42** [reply]
Ahhh, I had removed the `@livewireScriptConfig` line, but looks like that still needs to be there. My layout now has the following in it...

```
    @fluxScripts
    @livewireScriptConfig
```

And it's all working. Thank you.

**Reply by jeffchown** [reply]
Great.  You're welcome, @rikh42
